### DISCUSSION TITLE
Sizes of input arguments do not match

### DISCUSSION THREAD

**Discussion by zhaozzt** [open-edge-platform/anomalib#1844] [Q&A]
![4](https://github.com/openvinotoolkit/anomalib/assets/127210381/2f263141-9577-4447-b8ac-90311810000d)

**Comment by zhaozzt** [comment]
helpme

**Comment by zhaozzt** [comment]
helpme

**Comment by zhaozzt** [comment]
HELP ME!!!!!   I DONT KNOW

**Comment by blaz-r** [comment]
Hi, please share more of the error message. Also this seems to be a duplicate of #1843.

**Reply by zhaozzt** [reply]
this is details
![9](https://github.com/openvinotoolkit/anomalib/assets/127210381/723e3bc1-aef1-4805-a363-b7b0b5de74ea)
![10](https://github.com/openvinotoolkit/anomalib/assets/127210381/132770eb-2d4b-44f6-b4e7-2b409b2047f9)
![11](https://github.com/openvinotoolkit/anomalib/assets/127210381/d14394ab-9ddf-4fef-9444-66df67801ba4)
![12](https://github.com/openvinotoolkit/anomalib/assets/127210381/e605a03a-0bfd-4567-b253-f4d50373466d)
